packages/web-react/src/components/Card/stories/CardBody.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { Button } from '../../Button';
+import { Container } from '../../Container';
+import { PartnerLogo } from '../../PartnerLogo';
+import Card from '../Card';
+import CardBody from '../CardBody';
+import CardEyebrow from '../CardEyebrow';
+import CardFooter from '../CardFooter';
+import CardLogo from '../CardLogo';
+import CardMedia from '../CardMedia';
+import CardTitle from '../CardTitle';
+import { LOGO, MEDIA_IMAGE } from '../demo/constants';
+import ReadMe from '../README.md';
+
+const meta: Meta<typeof CardBody> = {
+  title: 'Components/Card',
+  component: CardBody,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+      description: 'Text to display in the CardBody.',
+      table: {
+        defaultValue: { summary: 'CardBody Text' },
+      },
+    },
+  },
+  args: {
+    children: 'CardBody Text',
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof CardBody>;
+
+export const CardBodyComponent: Story = {
```

Comment:
```suggestion
export const CardBodyPlayground: Story = {
```

so the URL is the same as elsewhere